website/content/en/docs/ansible/reference/watches.md

```diff
@@ -28,7 +28,8 @@ be monitored for updates and cached.
 * **vars**: This is an arbitrary map of key-value pairs. The contents will be
   passed as `extra_vars` to the playbook or role specified for this watch.
 * **reconcilePeriod** (optional): The reconciliation interval, how often the
-  role/playbook is run, for a given CR.
+  role/playbook is run, for a given CR. Note that a lower period will correct entropy more quickly, 
```

Comment:
The maximum interval in seconds that the operator will wait before beginning another reconcile, even if no watched events are received. When an operator watches many resources, each reconcile can become expensive, and a low value here can actually reduce performance, so it is generally recommended that this is only used when `watchDependentResources` is `False`.

Reply:
It should not really be used with the watches.